distutils/tests/unix_compat.py

```diff
@@ -1,10 +1,15 @@
+from __future__ import annotations
+
 import sys
+from types import ModuleType
 
+grp: ModuleType | None = None
+pwd: ModuleType | None = None
 try:
     import grp
     import pwd
 except ImportError:
-    grp = pwd = None
+    pass
```

Comment:
I'd be tempted to use `contextlib.suppress(ImportError)` to eliminate the no-op block... except that requires a prior import, which may lead to problems. Also, I give it a 50% chance that mypy recognizes that approach.